src/frontend/sass/splash-screen.scss

```diff
@@ -0,0 +1,44 @@
+@import './mat-colors';
+// Custom theme support
+@import './custom';
+app-root {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  color: $white;
+}
+
+body {
+  background-image: url(/assets/login-bg.jpg);
+  margin: 0;
+  padding: 0;
+}
+
+.splash-screen-loader {
+  border: 16px solid $white-87-opacity;
```

Comment:
Can you follow the `*.theme.scss` mixin based pattern we have elsewhere. Look at `boolean-indicator.component.theme.scss`for a simple example. Only the styling that affects theming should be in the theme file, everything else should be in`splash-screen.scss`.

Reply:
We follow the angular material theming guide that can be found here: https://material.angular.io/guide/theming-your-components